pm_icecon/bt/params/amsr2.py

```diff
@@ -1,84 +1,67 @@
-"""amsr2.py: bootstrap parameters file.
```

Comment:
I'd like to add more information about when these parameters are most appropriately used!